sunpy/tests/figure_hashes_mpl_332_ft_261_astropy_403.json

```diff
@@ -22,7 +22,7 @@
     "sunpy.map.tests.test_plotting.test_plot_masked_aia171_nowcsaxes": "69e28235fa42379a94f7da6cc39b375f361d1e26dc0f94e6b8c5f05894cc64c7",
     "sunpy.map.tests.test_plotting.test_plot_masked_aia171_superpixel": "154586d5635f895b5f9170353f70f427e23ab0be4ccabf743e668864653f54e3",
     "sunpy.map.tests.test_plotting.test_plot_masked_aia171_superpixel_nowcsaxes": "ec4cfe2687ea0a1f62f1a80f07e69ef5a89ef5aa2dbebc58599fb16b98ebafcb",
-    "sunpy.map.tests.test_plotting.test_plot_rotated_aia171": "878fedb8b890018434ca66aa09b8aacc5c2bd6a39b4cbefed36121998aca2bcc",
+    "sunpy.map.tests.test_plotting.test_plot_rotated_aia171": "06ed44d872a6e1e5ec2dd6a522ee81f68a2c8420dea1bc7491bc64d9e3d3e147",
```

Comment:
ooi, what changed?

Reply:
A few pixels, nothing major. I think due to the precision at which the auxillary information is stored in the FITS header.